### PR TITLE
init(rawkode.studio): step one ❤️

### PR DESCRIPTION
**Pull request by icepuma**

### CODE REVIEW

**Comment by icepuma** [comment]
@rawkode could you please do the necessary magic to fix:
```
  ✘ [ERROR] A request to the Cloudflare API (/accounts/***/pages/projects/rawkode-studio) failed.
    Project not found. The specified project name does not match any of your existing projects. [code: 8000007]
```

**Comment by rawkode** [comment]
Done

**Review by rawkode — APPROVED** [review]

**Comment by rawkode** [comment]
Merge if you're happy

File: projects/rawkode.studio/.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ZITADEL_CLIENT_ID="293097955970320066"
```
[the remark this answers is not in the document]
Comment (icepuma) [reply projects/rawkode.studio/.env:1]: That's just a copy for now. Will implement this properly when I have something to hide behind login :D

**Pull request merged**